.github/workflows/frontend_ci.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Lint
         run: yarn lint
 
+      - name: Unitary Tests
```

Comment:
I've never seen it called this either `Unit tests` or better just `Frontend tests`

Reply:
Changed stage name